index.bs

```diff
@@ -1258,16 +1258,16 @@ the {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}} methods.
 </pre>
 
 <div dfn-type="enum-value" dfn-for="Transport">
-    Authenticators may communicate with Clients using a variety of transports. This enumeration defines a hint as to how Clients
-    might communicate with a particular Authenticator in order to obtain an assertion for a specific credential. Note that these
-    hints represent the [=[RP]=]'s best belief as to how an Authenticator may be reached. A [=[RP]=] may obtain a list of
-    transports hints from some [=attestation statement formats=] or via some out-of-band mechanism; it is outside the scope of
+    Authenticators MAY communicate with Clients using a variety of transports. This enumeration defines a hint as to how Clients
+    might communicate with a particular Authenticator in order to obtain an assertion for a specific credential.  Note that these
+    hints represent the [=[RP]=]'s best belief as to how an Authenticator MAY be reached.  A [=[RP]=] MAY obtain a list of
+    transports hints from some attestation statement formats or via some out-of-band mechanism; it is outside the scope of
     this specification to define that mechanism.
 
     <ul>
-        <li><dfn>usb</dfn> - the respective Authenticator may be contacted over USB.
-        <li><dfn>nfc</dfn> - the respective Authenticator may be contacted over Near Field Communication (NFC).
-        <li><dfn>ble</dfn> - the respective Authenticator may be contacted over Bluetooth Smart (Bluetooth Low Energy / BLE).
+        <li><dfn>usb</dfn> - the respective Authenticator MAY be contacted over USB.
+        <li><dfn>nfc</dfn> - the respective Authenticator MAY be contacted over Near Field Communication (NFC).
+        <li><dfn>ble</dfn> - the respective Authenticator MAY be contacted over Bluetooth Smart (Bluetooth Low Energy / BLE).
```

Comment:
This is describing capabilities.  Please change all three of these to "can".

Reply:
done.